core/controlplane/config/templates/stack-template.json

```diff
@@ -1488,6 +1488,11 @@
     "StackName": {
       "Description": "The name of this stack which is used by node pool stacks to import outputs from this stack",
       "Value": { "Ref": "AWS::StackName" }
+    },
+    "ControllerIAMRoleArn": {
+      "Description": "The ARN of the IAM role for this Node Pool",
```

Comment:
there is a typo, The ARN of the IAM role for this Node Pool should be The ARN of the IAM role for the Controllers